docs/blazor-scenarios.md

```diff
@@ -0,0 +1,22 @@
+
+# Blazor Scenarios
+
+## New Blazorwasm Template Size On Disk
+An introduction of how to run scenario tests can be found in [Scenarios Tests Guide](link).  The current document has specific instruction to run blazor scenario tests. 
+### Step 1 Initialize Environment
+Same instruction of [Step 1 in Scenario Tests Guide](scenarios-workflow.md).
+### Step 2 Run Precommand
+```
+cd blazor
+py -3 pre.py publish --msbuild "/p:_TrimmerDumpDependencies=true"
+```
+The above command creates a new blazorwasm template in `blazor\app\` and publish it to `blazor\pub\`. The `--msbuild "/p:_TrimmerDumpDependencies=true"` argument is optional and can be specified to generate linker dump from the build, which is saved to `blazor\traces\`. 
+
+### Step 3 Run Test
+```
+py -3 test.py sod --scenario-name "SOD - New Blazor Template - Publish"
+```
+This runs the test. Note `--scenario-name` is optional.
+### Step 4 Run Postcommand
+Same instruction of [Step 4 of Scenario Tests Guide](scenarios-workflow.md).
```

Comment:
Following these steps it is kind of a hassle to have to click though to another document to do a step. Is there some way we can embed this bit in the document, or at least make the link hit an anchor in the markdown do that I don't have to scroll around to find what I want.

Reply:
Good point! I didn't find how to embed a paragraph from another document. Adding some anchors sounds good.